lib/msf/core/session_manager.rb

```diff
@@ -217,15 +217,6 @@ def register(session)
       # Insert the session into the session hash table
       self[next_sid.to_i] = session
 
-      # Notify the framework that we have a new session opening up...
-      # Don't let errant event handlers kill our session
-      begin
```

Comment:
We'll need a version of this here to continue to handle other types of sessions (e.g. shell).